<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionSettingsDialog</class>
 <widget class="QDialog" name="ConnectionSettingsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="connectionCancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connectionSaveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButton_1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Auto selection</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Wallet will connect to local daemon on default port. In case of no local daemon running it will launch built-in node.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>Embedded</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Built-in embedded node will be used.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QRadioButton" name="radioButton_3">
       <property name="text">
        <string>Local daemon</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Port: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="m_localDaemonPort">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>18322</number>
         </property>
         <property name="displayIntegerBase">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Wallet will connect to local daemon on specified port.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QRadioButton" name="radioButton_4">
       <property name="text">
        <string>Remote daemon</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QComboBox" name="remoteNodesComboBox">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::InsertAtTop</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addNodeButton">
         <property name="text">
          <string>Add node</string>
         </property>
         <property name="icon">
          <iconset resource="../../resources.qrc">
           <normaloff>:/icons/add</normaloff>:/icons/add</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeNodeButton">
         <property name="text">
          <string>Remove node</string>
         </property>
         <property name="icon">
          <iconset resource="../../resources.qrc">
           <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QLabel" name="label_5">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Wallet will connect to remote node running in local or global network. Please select node or specify IP address or domain name and port. In this mode wallet sends 0.25% fee from each transaction to the node, but no more than 10 BFR.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>connectionCancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ConnectionSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>549</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>connectionSaveButton</sender>
   <signal>clicked()</signal>
   <receiver>ConnectionSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>630</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addNodeButton</sender>
   <signal>clicked()</signal>
   <receiver>ConnectionSettingsDialog</receiver>
   <slot>addNodeClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeNodeButton</sender>
   <signal>clicked()</signal>
   <receiver>ConnectionSettingsDialog</receiver>
   <slot>removeNodeClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
